test/lib/linter/snapshots/linter.ts.md

```diff
@@ -446,41 +445,39 @@ Generated by [AVA](https://avajs.dev).
         messages: [
           {
             column: 17,
-            fatal: undefined,
             line: 47,
             message: 'Use of deprecated library \'sap.ui.commons\'',
             ruleId: 'ui5-linter-no-deprecated-library',
             severity: 2,
           },
           {
             column: 13,
-            fatal: undefined,
             line: 59,
             message: 'Use of deprecated property \'sap.ui5/resources/js\'',
-            ruleId: 'ui5-linter-no-deprecated-api',
+            ruleId: 'ui5-linter-no-deprecated-property',
             severity: 2,
           },
           {
             column: 21,
-            fatal: undefined,
             line: 72,
-            message: 'Use of deprecated property \'sap.ui5/models/odata-v4/settings/synchronizationMode\' of sap.ui.model.odata.v4.ODataModel',
+            message: 'Use of deprecated property \'synchronizationMode\' of class \'sap.ui.model.odata.v4.ODataModel\'',
+            messageDetails: 'As of Version 1.110.0, this parameter is obsolete.',
             ruleId: 'ui5-linter-no-deprecated-api',
             severity: 2,
           },
           {
             column: 21,
-            fatal: undefined,
             line: 78,
-            message: 'Use of deprecated property \'sap.ui5/models/odata-v4-via-dataSource/settings/synchronizationMode\' of sap.ui.model.odata.v4.ODataModel',
```

Comment:
It seems that the path to the dataSource got lost and only the property is left.
Was this intentional?
I'm fine with it as the positiinalInfo (line/column) is correct

Reply:
Good catch. The message was better before.